apps/teams-test-app/public/manifest.json

```diff
@@ -1,10 +1,52 @@
 {
-  "short_name": "Teams Test App",
-  "name": "Teams Test App",
-  "icons": [
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.16/MicrosoftTeams.schema.json",
+  "version": "1.0.0",
+  "manifestVersion": "1.16",
+  "id": "a79a2374-ec2a-486e-a3dc-b913fdb1f365",
+  "packageName": "com.package.name",
+  "name": { "short": "M365 Test App", "full": "" },
```

Comment:
Maybe add a full name too just in case that becomes a validation check in the future. :D